src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java

```diff
@@ -25,8 +23,15 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.UUID;
```

Comment:
Why replace `import org.gridsuite.filter.expertfilter.expertrule.StringExpertRule;` with `import org.gridsuite.filter.expertfilter.expertrule.*;` ?

and just after  replace `import java.util.*; `with
```
import java.util.ArrayList;
import java.util.Date;
import java.util.List;
import java.util.Set;
import java.util.TreeSet;
import java.util.UUID;
```
?

Reply:
modified